upgrade-notes/storefront_20240731_062725.md

```diff
@@ -0,0 +1,3 @@
+#### management of delivery addresses ([#3290](https://github.com/shopsys/shopsys/pull/3290))
+
+-   customer can create and edit delivery address in the profile
```

Comment:
This is an interesting point to mention, but what you should not omit are things like `customer type was modified` (this is important). Maybe you'll come up with something more...

Reply:
Thanks for a good tip, I added some more info :)